src/compiler/moduleNameResolver.ts

```diff
@@ -1576,6 +1576,20 @@ namespace ts {
 
     /** Return the file if it exists. */
     function tryFile(fileName: string, onlyRecordFailures: boolean, state: ModuleResolutionState): string | undefined {
+        const ext = tryGetExtensionFromPath(fileName) ?? "";

```

Comment:
Please rewrite this so that there is no path manipulation when moduleSuffixes is not specified.

Reply:
Fixed